tests/functional/tests_user_change_pass.py

```diff
@@ -0,0 +1,156 @@
+"""Testing ability to change password with existing users
+For getting valid response we need token of that user, old password and new password"""
+
+from tests.constants.constants import UserToTest, InvalidValues
+from tests.functional import ApiTestBase
+
+
+class TestChangePass(ApiTestBase):
```

Comment:
All test cases in this class are using exactly the same steps, please take a look to DDT python module
https://ddt.readthedocs.io/en/latest/example.html